CMakeLists.txt

```diff
@@ -711,11 +693,11 @@ if(${PLAYER_AUDIO_BACKEND} MATCHES "^SDL2.*$")
 		else()
 			message(STATUS "MIDI playback: WildMidi")
 		endif()
-	elseif(${PLAYER_ENABLE_FMMIDI} MATCHES "ON")
+	elseif(PLAYER_ENABLE_FMMIDI)
 		message(STATUS "MIDI playback: built-in (FmMidi)")
 	elseif(${PLAYER_ENABLE_FMMIDI} MATCHES "Fallback")
 		message(STATUS "MIDI playback: SDL2_mixer. Fallback to built-in (FmMidi)")
-	elseif(${PLAYER_ENABLE_FMMIDI} MATCHES "OFF")
+	elseif(NOT PLAYER_ENABLE_FMMIDI)
```

Comment:
because that's the last elseif this could be changed to else(), no idea why I wrote it that way.